dist/index.js

```diff
@@ -388,8 +388,8 @@ async function run() {
       waitForMinutes = MAX_WAIT_MINUTES;
     }
 
-    const forceNewDeployInput = core.getInput('force-new-deployment', { required: false }) || false;
-    const forceNewDeployment = forceNewDeployInput != undefined && (forceNewDeployInput.toLowerCase === 'true' || forceNewDeployInput);
+    const forceNewDeployInput = core.getInput('force-new-deployment', { required: false }) || 'false';
```

Comment:
Why do we need `|| 'false'` condition? Can't we just have the same logic as it is done for the **wait-for-service-stability** parameter?

```
const forceNewDeployInput = core.getInput('force-new-deployment', { required: false });
const forceNewDeployment = forceNewDeployInput.toLowerCase() === 'true';
```

Reply:
Technically it's not necessary, but I find it easier to read. This is because `core.getInput('force-new-deployment', { required: false})` returns an empty string if it's not defined. To a reader who isn't familiar with the API, this makes sure they understand that if it isn't defined, the response should still be handled as a string rather than as an `undefined` or `null` value.

If you need it removed to merge the PR I'll remove it, but I find an explicit string value of `'false'` cleaner than an expectation that the `core.getInput()` function returns an empty string when the input is not provided.

Reply:
@yehudacohen Great. I guess that extra condition works for me. Thank you for fixing the issue.